packages/ignition/src/app/locales/pt-br.json

```diff
@@ -42,18 +42,21 @@
   "states": "Estados",
   "show-all": "Todos os ícones",
   "icon": "ícone",
+  "learn-more": "Saiba mais",
   "table": {
-    "headers": ["Classes", "Propriedades"]
+    "headers": ["Classes", "Propriedades"],
+    "show-all": "Mostrar mais",
```

Comment:
I am curious on why "show more" is all lowercase and "Mostrar mais" is capitalized.